chains/solana/Makefile

```
@@ -44,6 +48,9 @@ format:
 go-tests:
 	go test -v ./... -json -covermode=atomic -coverpkg=./... -coverprofile=integration_coverage.txt 2>&1 | tee /tmp/gotest.log | gotestloghelper -ci=true -singlepackage=true -hidepassingtests=false -hidepassinglogs=false
 
-.PHONY: generate-idl
-generate-idl:
+.PHONY: build-contracts
+build-contracts:
 	cd ./contracts && anchor build
+
+.PHONY: solana-checks
+solana-checks: clippy anchor-go-gen format gomodtidy lint-go rust-tests go-tests build-contracts
```

The `anchor-go-gen` is already building the contracts, could you please remove that redundancy to bring down the time it takes to run this command?
```suggestion
solana-checks: clippy anchor-go-gen format gomodtidy lint-go rust-tests go-tests
```

---

No, I needed it to recreate the IDL, as the tests modify them adding the program_ids